src/content/docs/extensions/vector.mdx

```diff
@@ -0,0 +1,270 @@
+```
+
+### Create the Vector Index
+
+Creating a new vector index as follows:
+```cypher
+CALL CREATE_VECTOR_INDEX(
+    'table_name',      // Name of the table containing the vector column
+    'index_name',      // Name to identify the vector index
+    'column_name',     // Name of the column containing vector embeddings
+    [option_name := option_value]  // Optional parameters for index configuration
+);
+```
+
+:::note
+For optional parameters, we use `:=` instead of `=` to specify values.
+:::
+
+We support following options during index creation.
+| option name | description | default |
+| ------------ | ------------ | ---------- |
+| mu                | max degree of nodes in the upper graph. | 30 |
```

Comment:
- Explicitly mention that we're building a two layer HNSW.
- Explain a bit more that enlarging mu and ml can lead to better accuracy but slower index construction.

Reply:
Yeah we don't expect user to understand these parameters. I would be good to ducument the algorithm we are using and the effect of these parameters. We could do this later.